src/compas/geometry/frame.py

```diff
@@ -190,6 +190,10 @@ def zaxis(self):
             self._zaxis = self.xaxis.cross(self.yaxis)
         return self._zaxis
 
+    def axes(self):
```

Comment:
Hey @romanarust Just realized, could you also add this one in the docstring on top of class?

Reply:
added `axes` to the `Box` as well for convenience